hoomd/mpcd/SphereGeometry.h

```diff
@@ -85,23 +90,30 @@ class __attribute__((visibility("default"))) SphereGeometry
              * The perpendicular and parallel components of the velocity are:
              * v_perp = (v.n)n = (v.r/R^2)r
              * v_para = v-v_perp
+	     * V_vec is vector component of V(interface velocity)
              */
+	    
+	    const Scalar3 V_vec = m_V*pos/m_R;
```

Comment:
This is not quite right because when `dt > 0`, the particle will not be at R (the radius at the end of the streaming step). Assuming the solution above was perfect, what we want is:
```suggestion
	    const Scalar3 V_vec = m_V*pos/(m_R-m_V*dt);
```

Reply:
Thank you , In the back of my mind I was thinking that code was backtracking and putting back the particles at R, but now I get it that it's R'(R - V*dt) where particle collided with the interface.